deploy/kustomize/overlays/downstream/hco_cr.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: hco.kubevirt.io/v1alpha1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
```

Comment:
IIRC I don't think things like this (ie. downstream) are supposed to be included in the upstream source.

Reply:
Agreed.
So in GitHub we'll keep only upstream-related resources, and downstream repo will contain the downstream-related resources?